ingestion_tools/scripts/data_validation/tests/annotation/test_segmentationmask_annotation.py

```diff
@@ -0,0 +1,168 @@
+from typing import Dict
+
+import numpy as np
+import pytest
+from mrcfile.mrcinterpreter import MrcInterpreter
+from tests.helper_mrc import HelperTestMRCHeader
+
+ZATTRS_AXIS_ORDER = ["z", "y", "x"]
+
+
+# By setting this scope to session, scope="session" fixtures will be reinitialized for each run + voxel_spacing combination
+@pytest.mark.annotation
+@pytest.mark.parametrize("run_name, voxel_spacing", pytest.run_spacing_combinations, scope="session")
+class TestSegmentationMask(HelperTestMRCHeader):
```

Comment:
This class inherits the tests from HelperTestMRCHeader, which contains mrc header checks (this helper class is used across various entities that need mrc header validation checks)